### DISCUSSION TITLE
Meeting notes - May 30th, 2023

### DISCUSSION THREAD

**Discussion by Chris-ECE** [UNECE/GSBPM_GAMSO_Revision#13] [General]
Participants: Andrea Petres, Antti Santaharju, Christopher Jones, Cory Chobanik, Edgardo Greising, Elodie Kranklader, Florian Vucko, Gabriel Gamez, Giorgia Simeoni, InKyung Choi, Kevin McCormack, Nadia Mignolli, Zoltán Vereczkei

This call looked at the [Disseminate phase](https://github.com/UNECE/GSBPM_GAMSO_Revision/issues/4). 

Nadia Mignolli joined the team. – Welcome Nadia!

**General comments**
•	Regarding New Zealand’s comment that NSO’s might not disseminate final products, but allow others to do so. NZ suggested to change the wording to “ ….. there are more diverse user groups and that our data & products should be more interoperable and consumable ….”. **Zoltán agreed to update the document with the final wording.**

**Phase 7 Introduction**
•	The GeoGSBPM comment about cataloguing and tagging products was agreed.

**Sub-process 7.1 Update output systems**
•	Agreed to add a footnote on the difference between the product and output (to indicate some can be intermediate between stages in the process).
•	Discussed replacing the word “system” with module or component.
     o	Zoltán suggested renaming the phase sub-process to indicate that it is not the system itself being updated, but it’s content.
     o	Edgardo wasn’t sure about this.
     o	Andrea said this subprocess provides a connection between analysis and dissemination.
     o	Giorgia said that if we focus on content only, it could be harder to manage the difference with 7.2.
     o	Cory suggested this could be context specific, as there could be a pre-release area, and in 7.2 the data are turned into products. It might depend on the organization’s level of maturity.
     o	Zoltán said that in an ideal world, you should know what data formats you are going to use in advance, which would be an overarching process.
     o	InKyung suggested to leave this to the next revision of these standards.
•	Updating output systems. – Is this not already done in the Build phase?
     o	Zoltán suggested we can leave this.

**Sub-process 7.2. Produce dissemination products (also relevant to 7.3)**
•	Regarding whether the moment of dissemination takes place under 7.2 or 7.3:
     o	Zoltán understood that this happens in 7.3, but requires clearer text.
     o	Giorgia agreed.
     o	InKyung suggested renaming 7.3 to make it clearer, e.g. “release and manage dissemination products”.
     o	Nadia suggested removing the word “manage” from the title of 7.3.
          	InKyung mentioned the overarching processes within GSBPM that manage quality and metadata, so management is not exclusive to GAMSO.
          	Edgardo cited embargos and translations as being things that involve management within 7.3. He also pointed out that retracting a release could not be included in the tasks if the word manage is removed.
          	Tasks for this subprocess include translations, controls, and decisions. Andrea asked if “release” included transmission of data.
          	InKyung felt that the action verb at the start of the title doesn’t need to include every task within that sub-process.
          	Andrea, Giorgia and Gabriel supported removing the word manage.
     o	Zoltán said that from GSIM, a product is a package, which makes the phrase “dissemination product” a little confusing.
          	InKyung suggested a footnote under 7.2, we can explain that a product is a packaged output disseminated to users
          	Edgardo said the term “dissemination” helps to differentiate from intermediate products.
     o	Andrea asked about statistical outputs, and intermediate products. InKyung suggested allowing redundancy to reinforce the meaning of those terms.
•	Regarding the need for additional disclosure control, for example when maps are created:
     o	Gabriel said that GSBPM is not linear, and previous sub-processes could be revisited.
     o	**Decision**: It was suggested to add a sentence suggesting going back to 6.4 in the case of geospatial outputs.

**Sub-process 7.3. Manage release of dissemination products**
•	**Decision**: to rename the sub-process “Release dissemination products” (see above discussion)
•	Adding version numbers to products, rather than just retracting them in the event of an error.
     o	Zoltán suggested this could be a country-specific feature for those countries who desire it.
•	About discussion of dissemination planning
     o	InKyung noted dissemination is not included in the design phase.

**Sub-process 7.4. Promote dissemination products**
•	Regarding whether “manage user support” or “promote dissemination products” should be considered as overarching processes.
     o	There are phases where these items are not relevant, so it is not clear they are really overarching.
     o	Manage user support is specific to the product.
     o	Edgardo suggested talking of “product support”. – **This can be discussed in a later call.**

**Sub-process 7.5. Manage user support**
•	This could be renamed “Provide user support”.

**Comment by InKyungChoi** [comment]
To be done: final wording (“ ….. there are more diverse user groups and that our data & products should be more interoperable and consumable ….”.)
